business/workloads.go

```diff
@@ -613,6 +635,12 @@ func fetchWorkloads(ctx context.Context, layer *Layer, namespace string, labelSe
 		return nil, err
 	}
 
+	if cluster == "" {
+		cluster = kubernetes.HomeClusterName
+	}
```

Comment:
I don't think you need to have a special case for the empty string `""`, you can just pass the `kubernetes.HomeClusterName` where it hasn't been updated for multi-cluster yet.

Reply:
As the 'cluster' is a param coming from REST API, it can be empty or missing, for instance in called from E2E tests or any other REST Client. So adding default value here secures the backend itself.
Do you mean to add that default "HomeClusterName" value in UI calls?